src/main/java/org/acoli/conll/rdf/CoNLLRDFUpdater.java

```diff
@@ -288,15 +279,15 @@ private void unloadBuffer(Triple<List<String>, String, List<String>> sentBufferT
 		 * 			- total no. of iterations
 		 * 			- total time
 		 */
-		private List<Pair<Integer, Long>> executeUpdates(List<Triple<String, String, String>> updates) { 
+		private List<Pair<Integer, Long>> executeUpdates(List<Triple<String, String, String>> updates) {
 
 			String sent = new String();

```

Comment:
I detect that this code is problematic. According to the [Performance (PERFORMANCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#performance-performance), [Dm: Method invokes inefficient new String() constructor (DM_STRING_VOID_CTOR)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#dm-method-invokes-inefficient-new-string-constructor-dm-string-void-ctor).
Creating a new java.lang.String object using the no-argument constructor wastes memory because the object so created will be functionally indistinguishable from the empty string constant "".  Java guarantees that identical string constants will be represented by the same String object.  Therefore, you should just use the empty string constant directly.